server/src/main/java/org/spine3/server/event/EventBus.java

```diff
@@ -386,6 +449,30 @@ public EventStore getEventStore() {
         }
 
         /**
+         * Specifies an {@code Executor} for returning event stream from {@code EventStore}.
+         *
+         * <p>This {@code Executor} instance will be used for creating
+         * new {@code EventStore} instance when building {@code EventBus}, <em>if</em>
+         * {@code EventStore} was not explicitly set in the builder.
+         *
+         * <p>If an {@code Executor} is not set in the builder, {@link MoreExecutors#directExecutor()}
+         * will be used.
+         *
+         * @see #setEventStore(EventStore)
+         */
+        @SuppressWarnings("MethodParameterNamingConvention")
+        public Builder setEventStoreStreamExecutor(Executor eventStoreStreamExecutor) {
+            checkState(eventStore == null, EVENT_STORE_CONFIGURED_MESSAGE);
```

Comment:
Codecov browser plugin shows that `null` branches are probably not tested. Please check.